platform/android/src/annotation/polygon.cpp

```diff
@@ -50,12 +50,18 @@ float Polygon::getOpacity(jni::JNIEnv& env, jni::Object<Polygon> polygon) {
 
 mbgl::Color Polygon::getFillColor(jni::JNIEnv& env, jni::Object<Polygon> polygon) {
     static auto field = Polygon::javaClass.GetField<int>(env, "fillColor");
-    return *conversion::convert<mbgl::Color, int>(env, polygon.Get(env, field));
+    auto jColor = polygon.Get(env, field);
+    mbgl::Color color =  *conversion::convert<mbgl::Color, int>(env, jColor);
+    jni::DeleteLocalRef(env, jColor);
```

Comment:
jColor is of the type jint and these are just aliases for normal native value types, so there are no special lifetime or cleanup considerations.